cmd/kubeconfig/admin.go

```diff
@@ -37,20 +35,21 @@ func kubeConfigAdminCmd() *cobra.Command {
 	$ kubectl get nodes`,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			c := CmdOpts(config.GetCmdOpts())
+			c.Logger = util.CLILogger()
 			if util.FileExists(c.K0sVars.AdminKubeConfigPath) {
 				content, err := ioutil.ReadFile(c.K0sVars.AdminKubeConfigPath)
 				if err != nil {
-					log.Fatal(err)
+					c.Logger.Fatal(err)
 				}
 
 				clusterAPIURL, err := c.getAPIURL()
 				if err != nil {
-					return fmt.Errorf("failed to fetch cluster's API Address: %w", err)
+					c.Logger.Fatalf("failed to fetch cluster's API Address: %v", err)
```

Comment:
Are you intentionally relying on `logrus.Fatalf()` exiting the process instead of returning `error` to the caller?  There are a number of these changes in this latest diff.

Reply:
I adopted this logic from https://github.com/k0sproject/k0s/blob/main/pkg/backup/manager.go#L117
I thought it makes the code clearer, and applied it to cases where we actually need the command to fail.

Reply:
I would rather keep returning error, because cobra handling it by it's own, by rendering help and etc.
Plus in general log.Fatal is dangerous practice (for example it doesn't respect defers).